thrift/interfaces.go

```diff
@@ -20,27 +20,73 @@
 
 package thrift
 
-import athrift "github.com/apache/thrift/lib/go/thrift"
+import (
+	"fmt"
 
-// This file defines interfaces that are used or exposed by thrift-gen generated code.
-// TChanClient is used by the generated code to make outgoing requests.
-// TChanServer is exposed by the generated code, and is called on incoming requests.
+	athrift "github.com/apache/thrift/lib/go/thrift"
+)
 
-// TChanClient abstracts calling a Thrift endpoint, and is used by the generated client code.
+// This file defines interfaces that are used or exposed by thrift-gen
+// generated code. TChanClient is used by the generated code to make
+// outgoing requests. TChanServer is exposed by the generated code,
+// and is called on incoming requests.
+
+// TChanClient abstracts calling a Thrift endpoint, and is used by the
+// generated client code.
 type TChanClient interface {
-	// Call should be passed the method to call and the request/response Thrift structs.
-	Call(ctx Context, serviceName, methodName string, req, resp athrift.TStruct) (success bool, err error)
+	// Call should be passed the method to call and the
+	// request/response Thrift structs.
+	Call(
+		ctx Context, serviceName, methodName string, req, resp athrift.TStruct,
+	) (success bool, err error)
```

Comment:
please revert all whitespace changes (I don't want to lose useful information in `git blame` about when and why a specific line was changed).

We don't enforce any 80 character limits in the tchannel-go codebase -- we tend to use ~100 chars for comments, but there's no hard limit.